### DISCUSSION TITLE
New data provider ? (Alpaca)

### DISCUSSION THREAD

**Discussion by AdyGtl** [OpenBB-finance/OpenBB#6558] [Ideas]
Hello, 

While looking for APIs for options, I came accross Alpaca. Apparently they have recently developped a new API for option chains:
https://docs.alpaca.markets/reference/optionchain

I don't know to what extent adding new data providers to OpenBB is relevant but I share the information.

**Comment by Denis254-bit** [comment]
**`I have also come across an extension like the one you are speaking about and I think that they are new codes for a particular application that was already in use. `**
